control-plane-operator/controllers/hostedcontrolplane/manifests/kas.go

```diff
@@ -10,6 +10,26 @@ import (
 	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
 )
 
+const (
+	// KubeconfigScopeLabel is used to indicate the usage scope of the kubeconfig
+	KubeconfigScopeLabel = "hypershift.openshift.io/kubeconfig"
+)
+
+const (
+	// KubeconfigScopeExternal means the kubeconfig is for use by cluster-external
+	// clients
+	KubeconfigScopeExternal KubeconfigScope = "external"
+
+	// KubeconfigScopeLocal means the kubeconfig is for use by cluster-local
+	// clients (e.g. the service network)
+	KubeconfigScopeLocal KubeconfigScope = "local"
+
+	// TODO: What's this one for?
```

Comment:
This is the kubeconfig passed via ignition to worker nodes so they can bootstrap

Reply:
Thanks, documented